backend/handlers/post_collection_handler.py

```diff
@@ -75,7 +74,7 @@ def post(self, user):
                         user.key.urlsafe(),
                         entity_type,
                         post.key.urlsafe(),
-                        current_institution
+                        user.current_institution
```

Comment:
Ao responder um comentário:
500 : Error! get expected at least 1 arguments, got 0
 File "/home/mayza/workspace/web/backend/service_messages.py", line 19, in create_message
    institution_name = current_institution.get().name
TypeError: get expected at least 1 arguments, got 0